rotkehlchen/tests/fixtures/globaldb.py

```diff
@@ -68,6 +68,12 @@ def fixture_empty_global_addressbook() -> bool:
     return False
 
 
+@pytest.fixture(name='remove_global_assets')
+def fixture_remove_global_assets() -> list:
+    """Assets to remove from the globalDb"""
+    return []
```

Comment:
Typing here. Also this does not remove assets but identifier.

```suggestion
@pytest.fixture(name='remove_global_assets')
def fixture_remove_global_assets() -> list[str]:
    """Asset identifiers to remove from the globalDB"""
    return []
```

If typing does not work with empty list you can return none and make it return `list[str] | None` and do a falsy check in the test fixture you use it